nix/requirements_override.nix

```diff
@@ -108,6 +108,11 @@ in skipOverrides {
   };
 
   "libmozdata" = self: old: {
+    # Dual ESR Patch
+    src = pkgs.fetchurl {
```

Comment:
we should pin this down to this url using ``requirements.txt``. you should add the following line there
```
https://github.com/La0/libmozdata/archive/06a47b7f3f8e4a9485fe5611b43f3ad0eab76c8c.tar.gz#egg=libmozdata
```

and rerun ``make update APP=...``